src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/NuGetDeepLinkErrorView.xaml

```diff
@@ -0,0 +1,28 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.NuGetDeepLinkErrorView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             MaxWidth="700"
+             xmlns:local="clr-namespace:NuGet.PackageManagement.UI"
+             mc:Ignorable="d" 
+             d:DesignHeight="200" d:DesignWidth="1900">
+ 
+  <Grid>
+    <TextBlock Name="Message"
+               Margin="15,15,15,0"
+               FontSize="12"
+               TextWrapping="Wrap"
+               Text="{Binding ErrorWindowMessage}">
+    </TextBlock>
+    <Button Width="80"
```

Comment:
Since the name of the button is variable and can be supplied by the caller, do we maybe not want a fixed Width? Maybe a MinWidth would be better.